scripts/cp_artifacts.sh

```diff
@@ -20,11 +20,12 @@ if [[ -f "${SRC}/mattermost-desktop-${VERSION}-win-x64.zip" ]]; then
     cp "${SRC}/mattermost-desktop-${VERSION}-win-x64.zip" "${DEST}/mattermost-desktop-${VERSION}-win64.zip"
     SOMETHING_COPIED=$(($SOMETHING_COPIED + 2))
 fi
-if [[ -f "${SRC}/mattermost-desktop-setup-${VERSION}-win.exe" ]]; then
-    echo "Copying win-no-arch\n"
-    cp "${SRC}/mattermost-desktop-setup-${VERSION}-win.exe" "${DEST}/"
-    SOMETHING_COPIED=$(($SOMETHING_COPIED + 4))
-fi
+# we are not supplying this since we supply the msi
+# if [[ -f "${SRC}/mattermost-desktop-setup-${VERSION}-win.exe" ]]; then
+#     echo "Copying win-no-arch\n"
+#     cp "${SRC}/mattermost-desktop-setup-${VERSION}-win.exe" "${DEST}/"
+#     SOMETHING_COPIED=`expr $SOMETHING_COPIED + 4`
```

Comment:
The commented lines do not correspond to the non commented previous version

Reply:
no, it matches with the one it generates now, but I think I'll remove them anyway